lib/index.js

```diff
@@ -149,7 +154,12 @@ function _addEvent({event, eventHash, genesis, ledgerNode}, callback) {
         }
       }];
       storage.events.update({eventHash, patch}, callback);
-    }]
+    }],
+    // manually emit a `block.add` event
+    emit: ['updateEvent', (results, callback) => bedrock.events.emit(
+      'bedrock-ledger-storage.block.add', {
+        blockHeight, ledgerNodeId: ledgerNode.id
+      }, callback)]
```

Comment:
I don't know what moving the `emit` was all about, so I'm just going to assume it's correct. :)

Reply:
I added a more descriptive comment.